src/modules/News/NewsList.tsx

```diff
@@ -0,0 +1,146 @@
+    errorMessage: state.news.error,
+  }));
+
+  const [activeButton, setActiveButton] = useState('ALL-button');
+
+  const clickedButtonHandler = useCallback(
+    (name: string) => {
+      setActiveButton(name + '-button');
+      getNewsList(name);
+    },
+    [getNewsList],
+  );
+
+  useEffect(() => {
+    getNewsList();
+  }, [getNewsList]);
+
+  const buttons = useMemo(
+    () => (
+      <Box mb={2}>
+        {['ALL', 'MFI', 'INF'].map(name => (
+          <NewsButton
+            name={name}
+            key={name}
+            type="button"
+            text={t('newsPage.' + name.toLowerCase())}
+            className={activeButton == name + '-button' ? 'selected' : ''}
+            onClick={() => clickedButtonHandler(name)}
+          ></NewsButton>
+        ))}
+      </Box>
+    ),
+    [t, activeButton, clickedButtonHandler],
+  );
+
+  const newsTiles: JSX.Element[] = useMemo(
+    () =>
+      newsList.map(({ title, datetime, source, shortBody, leadingPhoto, _id }) => (
+        <StyledLink to={_id} key={title} data-cy="news-tile-container" sx={{ margin: '25px 25px' }}>
+          <DetailsTile backgroundColor={theme.palette.background.paper} padding="25px 25px" marginTop={1}>
+            <Grid container direction="row" justifyContent="flex-start" alignItems="flex-start">
+              <Grid item style={{ display: 'flex', alignItems: 'center', marginRight: '25px' }}>
+                <Box
+                  component="img"
+                  sx={{
+                    height: 210,
+                    width: 300,
+                    objectFit: 'cover',
+                    borderRadius: '25px 0px 0px 25px',
+                  }}
+                  src={leadingPhoto}
+                />
+              </Grid>
+              <Grid
+                item
+                xs={7}
+                style={{
+                  overflow: 'hidden',
+                  textOverflow: 'ellipsis',
+                  display: '-webkit-box',
+                  WebkitLineClamp: '8',
+                  WebkitBoxOrient: 'vertical',
+                }}
+              >
+                <Paragraph color={theme.palette.secondary.dark} fontSize={20}>
+                  {title}
+                </Paragraph>
+                <Paragraph fontWeight={700} fontSize={15} color={theme.palette.primary.main} margin="5px 0px 5px">
+                  {` • ${moment(datetime).format('DD-MM-YYYY')} • ${source}`}
+                </Paragraph>
+                {shortBody.length != 0 && (
+                  <Paragraph fontWeight={400} fontSize={15} color={theme.palette.primary.dark}>
+                    {shortBody}
+                  </Paragraph>
+                )}
+              </Grid>
+            </Grid>
+          </DetailsTile>
+        </StyledLink>
+      )),
+    [
+      newsList,
+      theme.palette.background.paper,
+      theme.palette.secondary.dark,
+      theme.palette.primary.dark,
+      theme.palette.primary.main,
+    ],
+  );
+
```

Comment:
Other components have the same error message written. I think it's a good idea to create a single global one, but wouldn't it be better to do this on a separate branch since it will affect multiple components?